core/camel-main/src/main/java/org/apache/camel/main/MainConfigurationProperties.java

```diff
@@ -559,4 +573,12 @@ public MainConfigurationProperties withAdditionalRoutesBuilder(Class... builders
         addRoutesBuilder(builders);
         return this;
     }
+
+    /**
+     * Add an additional {@link LambdaRouteBuilder} object to the known list of builders.
+     */
+    public MainConfigurationProperties withAdditionalL(LambdaRouteBuilder builder) {
```

Comment:
Maybe its a bit too short name as the others are additoinalRouteBuilder and this may seem like a typo with that lower case l and then an upper case L

Reply:
this is likely to be a mistake :)